docs/docs/deploy-gatsby.md

```diff
@@ -275,15 +275,9 @@ In order to deploy your Gatsby project using [Now](https://zeit.co/now), you can
 
 `npm install -g now`
 
-2.  Install a node server package (such as `serve`, or `http-server`)
+2.  Run `now` at the root of your Gatsby project, this will upload your project, run the `build` script, and then your `start` script.
 
-`npm install --save serve`
-
-3.  Add a `start` script to your `package.json` file, this is what Now will use to run your application:
-
-`"start": "serve public/"`
-
-4.  Run `now` at the root of your Gatsby project, this will upload your project, run the `build` script, and then your `start` script.
+For alternate [Now](https://zeit.co/now) deployment options, check out [this](/docs/deploying-to-now) document.
```

Comment:
Nit pick, but could you use [this document] for the link?

Reply:
Haha, yes absolutely!